checker/src/main/java/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -371,6 +371,11 @@ public NullnessAnnotatedTypeFactory(BaseTypeChecker checker) {
         NULLABLE_ALIASES.forEach(annotation -> addAliasedTypeAnnotation(annotation, NULLABLE));
         POLYNULL_ALIASES.forEach(annotation -> addAliasedTypeAnnotation(annotation, POLYNULL));
 
+        // Add compatible annotations from alias file
+        addAliasedTypeAnnotation(NonNull.class);
+        addAliasedTypeAnnotation(Nullable.class);
+        addAliasedTypeAnnotation(PolyNull.class);
```

Comment:
Another issue is that this only supports aliased type annotations, not also aliases for declaration annotations.
Would it be better to have two options `-AaliasedTypeAnnos` and `-AaliasedDeclAnnos`?
Both files would have the same format as described in the comment above, just one calls `addAliasedTypeAnnotation` and the other `addAliasedDeclAnnotation`.
(`addAliasedDeclAnnotation` requires a `annotationToUse` argument of type `AnnotationMirror`; we should figure out why we need it and how we could create it. Maybe aliased declaration annotations need something else?)

Reply:
I haven't looked into `addAliasedDeclAnnotation` closely. 
```
@param alias the fully-qualified name of the alias annotation
@param annotation the fully-qualified name of the canonical annotation
@param annotationToUse the annotation mirror to use
```
Not sure why it need both `annotation` and `annotationToUse`. It look a bit more complicated to add. I will just do `-AaliasedTypeAnnos` for now.